### PR TITLE
HPCC-13601 Travis-CI

### PR DESCRIPTION
**Pull request by GordonSmith**
Add support for Travis-CI/GitHub integration.

Fixes HPCC-13601

Signed-off-by: Gordon Smith gordon.smith@lexisnexis.com

### CODE REVIEW

**Comment by GordonSmith** [comment]
@richardkchapman Please Review

**Comment by AttilaVamos** [comment]
Automated Smoketest Build: success
HPCC Start: OK
HPCC Stop: OK

File: .travis.yml [hunk 1]
```diff
@@ -0,0 +1,21 @@
+language: cpp
+
+compiler:
+  - gcc
+
+before_install:
+  - echo $LANG
+  - echo $LC_ALL
+  - sudo apt-get update && sudo apt-get install -y  g++ gcc make cmake bison flex binutils-dev libldap2-dev libcppunit-dev libicu-dev libxslt1-dev zlib1g-dev libboost-regex-dev libssl-dev libarchive-dev python2.7-dev libv8-dev openjdk-6-jdk libapr1-dev libaprutil1-dev libhiredis-dev
```
Comment (richardkchapman) [inline .travis.yml]: Doesn't this depend on the distro?

Comment (GordonSmith) [inline .travis.yml]: Yes - but its the distro that Travis-CI uses to do the build.

**Pull request merged**